.github/CODEOWNERS

```diff
@@ -20,12 +20,15 @@
 # Database Operations for database changes
 src/Sql/** @bitwarden/dept-dbops
 util/EfShared/** @bitwarden/dept-dbops
-util/Migrator/** @bitwarden/dept-dbops
```

Comment:
We should heep the dept-dbops team as codeowners of at least folders containing SQL scripts: 
- util/Migrator/DbScripts
- util/Migrator/DbScripts_finalization
- util/Migrator/DbScripts_transition
- util/Migrator/MySql